spec/e2e/id_property_spec.rb

```diff
@@ -73,7 +73,27 @@
       it 'will set the id_property after a session has been created' do
         node = Clazz.new
         expect(node).to respond_to(:the_id)
-        expect(Clazz.mapped_label.indexes[:property_keys].sort).to eq([[:the_id], [:uuid]])
+        expect(Clazz.mapped_label.indexes[:property_keys].sort).to eq([[:the_id]])
+      end
+    end
+
+    describe 'when having neo_id configuration' do
+      before do
+        before_session do
+          Neo4j::Config[:id_property] = :neo_id
+          stub_const('Clazz', Class.new do
+            include Neo4j::ActiveNode
```

Comment:
There's a helper for this:

```ruby
stub_active_node_class('Clazz`)
```

I would go with something more descriptive than `Clazz` though.  Maybe `NeoIdTest`